### DISCUSSION TITLE
organizzazione fisica - 4

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#83] [Esercizi]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/e74a2c8e-feff-41f4-98d6-2e95b2bc1b35)

**Comment by daromacarlo** [comment]
![Immagine 2024-01-23 160500](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/8136463e-cff9-4415-82f5-cf27672b579d)
![Immagine 2024-01-23 160511](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/c6a8a68b-7509-4684-97d6-cec24e57fd8e)

**Reply by alemilos** [reply]
In AM2 non dovrebbe essere 104 ? Se prendi la parte superiore di 207/2

**Reply by daromacarlo** [reply]
Si, hai ragione
